flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/inference/StaticArgument.java

```diff
@@ -144,6 +145,45 @@ public static StaticArgument table(
         return new StaticArgument(name, dataType, null, isOptional, enrichTableTraits(traits));
     }
 
+    /**
+     * Declares a model argument such as {@code f(m => myModel)} or {@code f(m => MODEL myModel))}.
+     *
+     * <p>By only providing a conversion class, the argument supports a "polymorphic" behavior. In
+     * other words: it accepts models with arbitrary schemas or types. For this case, a class
+     * satisfying the model's conversion requirements must be used.
```

Comment:
```suggestion
     * <p>By using this method, the argument supports a "polymorphic" behavior. In
     * other words: it accepts models with arbitrary schemas or types.
```
conversion class does not exist for models yet